examples/complete/main.tf

```diff
@@ -1,7 +1,46 @@
+module "vpc" {
+  source  = "cloudposse/vpc/aws"
+  version = "1.1.0"
+
+  namespace  = var.namespace
+  stage      = var.stage
+  name       = var.name
+  cidr_block = "172.16.0.0/16"
+}
+
+module "subnets" {
+  source  = "cloudposse/dynamic-subnets/aws"
+  version = "2.0.2"
+
+  availability_zones   = var.availability_zones
+  namespace            = var.namespace
+  stage                = var.stage
+  name                 = var.name
+  vpc_id               = module.vpc.vpc_id
+  igw_id               = module.vpc.igw_id
```

Comment:
these inputs will need to be updated too

Reply:
what do you mean updated?